Cargo.toml

```diff
@@ -61,4 +61,5 @@ clap = { version = "4.5.23", features = ["derive"] }
 quickcheck = "1.0.3"
 
 [[example]]
-name = "file-uploader"
+#name = "file_uploader"
+name = "object_prompt"
```

Comment:
This probably removes the file_uploader example? maybe this needs one example section per example program in the crate.